assembly-h2o/build.gradle

```diff
@@ -106,7 +106,7 @@ if (project.hasProperty("doExtend")) {
     task printExtendedJarLocation() {
         doLast {
             def fileName = getH2OJarName(getOrDownloadOriginalJar())
-            println(file("private/extended/${fileName}.jar"))
+            println(file("private/extended/${fileName}-${version}.jar"))
```

Comment:
I thought this was fixed by https://github.com/h2oai/sparkling-water/commit/095ee7ef242b7266bd37010f1cdb22eb1a6c0a93 but still not, just tried. We need to ensure the generated name of the extended jar does not include the version as that could break some existing users environments. and keep the `printExtendedJarLocation` as it was before